mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -425,6 +431,21 @@ void dispatchShouldWorkAfterRestartForOldKeyRegistration() throws Exception {
                 assertThatListenerReceiveOneEvent(listener);
             }
 
+            @Test
+            void dispatchedMessagesShouldSurviveARabbitMQRestart() throws Exception {
+                eventBusWithKeyHandlerNotStarted.startWithoutStartingKeyRegistrationHandler();
+                MailboxListener listener = newAsyncListener();
+                eventBusWithKeyHandlerNotStarted.register(listener, KEY_1);
+                Mono<Void> dispatch = eventBusWithKeyHandlerNotStarted.dispatch(EVENT, KEY_1);
+                dispatch.block();
+
```

Comment:
assert there is no event received here ?

Reply:
I don't think it's necessary as we explicitly declared an event bus with the key handler not started, so it can't dequeue messages.

If you really want it we can add something like : 
```
assertThatThrownBy(() -> assertThatListenerReceiveOneEvent(listener)).isInstanceOf(ConditionTimeoutException.class);
``` 
But it make the test slower as we have to wait for the timeout to be sure we will not receive the message.

Reply:
Sorry I did read the test carefully. It's fine with this version